From 5745309faaad28e61b50fd7d77e2c4653f050112 Mon Sep 17 00:00:00 2001
From: Apaczer <94932128+Apaczer@users.noreply.github.com>
Date: Thu, 18 Dec 2025 14:01:04 +0100
Subject: [PATCH] config.def.h: update default configuration

- don't swap OK/CANCEL buttons
- quit_on_close_content (CLI)
- don't show RESTART app in menu (broken)
- don't Confirm to Quit/Close (thus press twice)
---
 config.def.h | 12 ++++++++----
 1 file changed, 8 insertions(+), 4 deletions(-)

diff --git a/config.def.h b/config.def.h
index 6b0a1d5422..44db234e72 100644
--- a/config.def.h
+++ b/config.def.h
@@ -762,7 +762,7 @@
 #define DEFAULT_MENU_SHOW_CONFIGURATIONS true
 #define DEFAULT_MENU_SHOW_HELP true
 #define DEFAULT_MENU_SHOW_QUIT true
-#define DEFAULT_MENU_SHOW_RESTART true
+#define DEFAULT_MENU_SHOW_RESTART false
 #define DEFAULT_MENU_SHOW_REBOOT true
 #define DEFAULT_MENU_SHOW_SHUTDOWN true
 #ifdef HAVE_MIST
@@ -791,7 +791,7 @@
 
 #define DEFAULT_MENU_INSERT_DISK_RESUME true
 
-#define DEFAULT_QUIT_ON_CLOSE_CONTENT QUIT_ON_CLOSE_CONTENT_DISABLED
+#define DEFAULT_QUIT_ON_CLOSE_CONTENT QUIT_ON_CLOSE_CONTENT_CLI
 
 /* While the menu is active, supported drivers
  * will display a screensaver after SCREENSAVER_TIMEOUT
@@ -963,7 +963,11 @@
 #define DEFAULT_OVERLAY_DPAD_DIAGONAL_SENSITIVITY 80
 #define DEFAULT_OVERLAY_ABXY_DIAGONAL_SENSITIVITY 50
 
+#if defined(MIYOO)
+#define DEFAULT_MENU_SWAP_OK_CANCEL_BUTTONS false
+#else
 #define DEFAULT_MENU_SWAP_OK_CANCEL_BUTTONS true
+#endif
 #define DEFAULT_MENU_SWAP_SCROLL_BUTTONS false
 #define DEFAULT_MENU_SINGLECLICK_PLAYLISTS false
 #define DEFAULT_MENU_ALLOW_TABS_BACK true
@@ -974,8 +978,8 @@
 #define DEFAULT_ALL_USERS_CONTROL_MENU false
 #endif
 
-#define DEFAULT_CONFIRM_QUIT true
-#define DEFAULT_CONFIRM_CLOSE true
+#define DEFAULT_CONFIRM_QUIT false
+#define DEFAULT_CONFIRM_CLOSE false
 #define DEFAULT_CONFIRM_RESET true
 
 #define DEFAULT_LOG_TO_FILE false
-- 
2.45.2.windows.1

